docs/manual/parameters/index.ipynb

```diff
@@ -282,7 +283,7 @@
     "can call the `splat()` method like this:\n",
     "\n",
     "```mojo\n",
-    "GenericArray[Float64].splat(8, 0)\n",
+    "GenericArray[Float64](8, 0)\n",
```

Comment:
```suggestion
    "GenericArray[Float64].splat(8, 0)\n",
```

Reply:
We need to revert this line, this is not being called on SIMD; this is calling a static method declared above on LL277-278 demonstrating the use of the `Self` type.